source/fundamentals/builders/sort.txt

```diff
@@ -11,3 +11,320 @@ Sorts Builders
+      {"_id": 3, "letter": "a", "food": "maple syrup"}
+      ...
+   
+An important thing for you to keep in mind is that MongoDB does
+not guarantee sort order for documents that have fields with identical values.
+This means that ``ascending("letter")`` could have any of these
+documents in the first position: 
+
+.. code-block:: none
+
+   {"_id": 3, "letter": "a", "food": "maple syrup"}
+   {"_id": 5, "letter": "a", "food": "milk and cookies"}
+   {"_id": 2, "letter": "a", "donuts and coffee"}
+
+If you would like to have a guaranteed sort order 
+for documents that have fields with identical values
+you can specify additional fields to sort
+on in the event of a tie. 
+
+We can specify an ``ascending`` sort on ``"letter"`` followed
+by ``"_id"`` as follows:
+
+.. code-block:: java
+
+   import static com.mongodb.client.model.Sorts.ascending;
+
+   // <MongoCollection setup code here>
+   
+   collection.find().sort(ascending("letter", "_id"))
+
+The above code snippet returns the documents in the
+``sort_example`` collection in the following order: 
+
+.. code-block:: none
+
+   {"_id": 2, "letter": "a", "food": "donuts and coffee"}
+   {"_id": 3, "letter": "a", "food": "maple syrup"}
+   {"_id": 5, "letter": "a", "food": "milk and cookies"}
+   {"_id": 4, "letter": "b", "food": "coffee with sugar"}
+   {"_id": 1, "letter": "c", "food": "coffee with milk"}
+   {"_id": 6, "letter": "c", "food": "maple donut"}
+
+
+Descending
+~~~~~~~~~~
+
+To specify a descending sort, use the ``Sorts.descending()``
+static factory method. A descending sort returns your documents
+from largest to smallest on the field name specified.
+
+The following code snippet shows how to specify a ``descending``
+sort on the ``"_id"`` field: 
+
+.. code-block:: java
+   
+   import static com.mongodb.client.model.Sorts.descending;
+
+   // <MongoCollection setup code here>
+   
+   collection.find().sort(descending("_id"))
+
+
+The above code snippet returns the documents in the
+``sort_example`` collection in the following order: 
+
+.. code-block:: none
+
+   {"_id": 6, "letter": "c", "food": "maple donut"}
+   {"_id": 5, "letter": "a", "food": "milk and cookies"}
+   {"_id": 4, "letter": "b", "food": "coffee with sugar"}
+   ...
+
+An important thing for you to keep in mind is that MongoDB does
+not guarantee sort order for documents that have fields with identical values.
+This means that ``descending("letter")`` could have any of these
+documents in the first position: 
+
+.. code-block:: none
+
+   {"_id": 1, "letter": "c", "food": "coffee with milk"}
+   {"_id": 6, "letter": "c", "food": "maple donut"}
+
+If you would like to have a guaranteed sort order 
+for documents that have fields with identical values
+you can specify additional fields to sort
+on in the event of a tie. 
+
+The following code snippet shows how to specify a ``descending``
+sort on the ``"letter"`` field followed by the ``"_id"`` field: 
```

Comment:
For variable names and field names the monospace formatting is all that is necessary. For values, we use just double quotes. We should choose just one rather than both.

Reply:
Field names just get monospace. Values get just double quotes (no monospace).

Reply:
Thanks for the note. This is fixed in:  https://github.com/mongodb/docs-java/pull/59/commits/571a18405e071ceee10e09ddb89047dd73b93ee5

Going forward, I will give field names monospace, put values in quotes without monospace.